pkgs/build-support/docker/default.nix

```diff
@@ -434,6 +434,8 @@ rec {
         if [[ -n "$fromImage" ]]; then
           echo "Unpacking base image..."
           tar -C image -xpf "$fromImage"
+          # Do not import the base image configuration
+          rm -f image/*.json
```

Comment:
@lo1tuma Ok, the latest patch did this.

FYI, regarding the manifest.json and image config, I started to work a bit on them https://github.com/nlewo/nixpkgs-contrail/blob/master/jobset.nix#L40 since they are required to push the image to a v2 registry.
I will try to upstream these changes later.